pkg/util/oc/oc_struct.go

```diff
@@ -135,6 +143,14 @@ func (o OC) DeleteSecret(t test.TestHelper, ns string, name string) {
 	})
 }
 
+func (o OC) DeleteConfigMap(t test.TestHelper, ns string, name string) {
+	t.T().Helper()
+	o.withKubeconfig(t, func() {
+		t.T().Helper()
+		shell.ExecuteIgnoreError(t, fmt.Sprintf(`kubectl -n %s delete configmap %s`, ns, name))
+	})
+}
```

Comment:
Since you probably copy-pasted this from `DeleteSecret`, can you instead create a `DeleteResource(t, ns, kind, name)` function and call it from both `DeleteSecret` and `DeleteConfigMap`?

Reply:
Done